### ISSUE TITLE
Install EXWM outside emacs

### ISSUE THREAD

**Issue by heeh** [ch11ng/exwm#884]
I love EXWM and have no problem using it. 
However, it took a while when setting this up for the first time.
Is there a reason that we have to set EXWM within emacs?
Would it be possible to install EXWM in apt or pacman with emacs dependency? (e.g. lightdm, gdm)

**Comment by yusi1** [comment]
EXWM is just a library that is added onto Emacs, so installing it without Emacs wouldn't be possible, because it literally is Emacs.

But you could have a package for EXWM that installs Emacs as a dependency like what you said and configures EXWM in Emacs automatically.

**Comment by medranocalvo** [comment]
Yes it should be possible to create a package which includes EXWM and additional setup (for example, *.desktop files).  I believe Guix packages EXWM in such a way.

**Comment by maker2413** [comment]
I believe the guix package manager has packages for most of emacs packages. I haven't ever looked into how they manage this but I imagine it would possible to do something similar for the AUR. I think the only reason this hasn't been done is just do to the fact that if you have emacs config checked into a repository install exwm is just as simple as installing emacs and cloning down your config.
